### ISSUE TITLE
@gridsome/source-contentful: Transform Markdown input with remark plugin

### ISSUE THREAD

**Issue by jefer590** [gridsome/gridsome#445]
## Summary
In Contentful, the text field  can have many shapes and one of them is specially Markdown.

The current problem is that if I get markdown input from contentful and I use the remark plugin, the markdown doesn't get transformed into HTML to get "gridsome friendly" html

### Basic example (expected result)
1. Create the `content` field in Contentful (using the markdown editor)
2. Create a new entry and publish it
3. Try fetching the entry in the `__explorer` path with the content
4. the expected result is that the Markdown gets transformed into HTML using remark

### Motivation
Get more sources to support markdown "out of the box" or just a little tweaks

**Comment by khuumi** [comment]
This would be useful for other data types as well. For example if I have a JSON or YAML document auto-generated with markdown fields it would be nice if there was a way to specify somewhere in the code to get these ready to use with v-html.

**Comment by perpetualgrimace** [comment]
Yes! More broadly, since we can use GraphQL to return front matter, YAML, JSON, etc., as data, I was really surprised that there isn't something like a `MarkdownParse()` helper to use in Vue components.

I know this isn't apples to apples — and Gridsome isn't a CMS — but check out how [Kirby CMS](https://getkirby.com/docs/reference/templates/helpers/kirbytext) allows any field to be easily converted to markdown.

Please consider renaming/retagging/reprioritizing this issue. 🙏

**Comment by perpetualgrimace** [comment]
Oh wait, is this what the `v-html` prop does 😅

**Comment by snoopdouglas** [comment]
Even better would be to have Markdown processed at the data layer, such that the browser never processes the Markdown. I'm opening a separate issue for this now
